nevergrad/benchmark/experiments.py

```diff
@@ -960,26 +960,26 @@ def rocket(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
                             yield xp
 
 
-@registry.register
-def mixsimulator(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
-    """MixSimulator of power plants
-    Budget 20, 40, ..., 1600.
-    Sequential or 30 workers."""
-    funcs = [OptimizeMix()]
-    seedg = create_seed_generator(seed)
-    optims: tp.List[str] = get_optimizers("basics", seed=next(seedg))  # type: ignore
-
-
-    seq = np.arange(0, 1601, 20)
-    for budget in seq:
-        for num_workers in [1, 30]:
-            if num_workers < budget:
-                for algo in optims:
-                    for fu in funcs:
+try:  # Dirty try/except because MixSimulator is not up to date.
+    OptimizeMix()
+    @registry.register
+    def mixsimulator(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
```

Comment:
don't do this at the base level, there is a Exception you can send from within the experiment instead:
https://github.com/facebookresearch/nevergrad/blob/f3e6085b368fb31a0501d327951560a9afc287ac/nevergrad/benchmark/experiments.py#L1452-L1455
Or better yet, in this case, you should send it from inside the function, example:
https://github.com/facebookresearch/nevergrad/blob/master/nevergrad/functions/images/imagelosses.py#L113-L120